sdk/src/dlob/DLOB.ts

```diff
@@ -131,6 +133,72 @@ export class DLOB {
 		return true;
 	}
 
+	public handleOrderRecord(record: OrderRecord): void {
+		this.insertOrder(record.order, record.user);
+	}
+
+	public handleOrderActionRecord(record: OrderActionRecord): void {
+		if (isVariant(record.action, 'place')) {
+			return;
+		}
+
+		if (isVariant(record.action, 'trigger')) {
+			if (record.taker !== null) {
+				const takerOrder = this.getOrder(record.takerOrderId, record.taker);
+				if (takerOrder) {
+					this.trigger(takerOrder, record.taker);
+				}
+			}
+
+			if (record.maker !== null) {
+				const makerOrder = this.getOrder(record.makerOrderId, record.maker);
+				if (makerOrder) {
+					this.trigger(makerOrder, record.maker);
+				}
+			}
```

Comment:
can return at the end of this `if` statement, or make the other cases `else if`. same as the ones below too.

Reply:
yup